File: nowcoder/BC26_计算三角形的周长和面积/BC26.go
package main
 
import (
    "fmt"
    "math"
)
 
func main() {
    var a, b, c float64
    fmt.Scanf("%f %f %f", &a, &b, &c)
    sum := a + b + c
    /*海伦公式p=(a+b+c)/2,s=sqrt(p(p-a)(p-b)(p-c));*/
    p := sum / 2
    area := math.Sqrt(p * (p - a) * (p - b) * (p - c))
    fmt.Printf("circumference=%.2f area=%.2f", sum, area)
}

File: nowcoder/BC26_计算三角形的周长和面积/BC26.c
#include<stdio.h>
int main() {
    float a,b,c;
    float cir,area;
    scanf("%f %f %f",&a,&b,&c);
    cir = a + b + c;
    /*海伦公式p=(a+b+c)/2,s=sqrt(p(p-a)(p-b)(p-c));*/
    float p=(a+b+c)/2.00;
    area = sqrt(p*(p-a)*(p-b)*(p-c));
    printf("circumference=%.2f area=%.2f",cir,area);
}